mobility_games/games/wall_audio_player.py

```diff
@@ -60,22 +63,27 @@ def config_callback(self, config, level):
         return config
 
     def get_dist(self, msg):
-        self.dist = msg.data
-        print(self.dist)
-        if self.speed:
-            filtereddist =  max(min(3*self.dist, 20), 2)
-            self.rate = 1.0/filtereddist;
-            self.r = rospy.Rate(1.0/self.rate)
+        """
+        Code to run when distance is updated on the smart_wall_dist node.
+        """
+        self.dist = msg.data #get distance
+        print(self.dist) #print distance
+        if self.speed: #if modulating speed
+            filtereddist =  max(min(3*self.dist, 20), 2) #modulate based on this function (CHANGE THIS FUNCTION EVENTUALLY TO BE BASED ON MAXFREQDIST AND ALTSOUNDDIST)
```

Comment:
These are different, the relationship between the pitch and the distance shouldn't have a similar relationship to the rate and the distance.  The pitch needed to change exponentially in order for the user to tell the difference, whereas the rate doesn't really need to so much.